config/notifiers.go

```diff
@@ -67,6 +67,18 @@ var (
 		Fallback:  `{{ template "slack.default.fallback" . }}`,
 	}
 
+	// DefaultHipchatConfig defines default values for Hipchat configurations.
+	DefaultHipchatConfig = HipchatConfig{
+		NotifierConfig: NotifierConfig{
+			VSendResolved: false,
```

Comment:
Why false by default. Slack is quite similar and has it as true.
I don't think there's one right default, but consistency is important.